lib/src/main/java/com/orange/ods/compose/component/chip/OdsFilterChip.kt

```diff
@@ -0,0 +1,130 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.compose.component.chip
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.ChipDefaults
+import androidx.compose.material.ChipDefaults.LeadingIconOpacity
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.FilterChip
+import androidx.compose.material.Icon
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.material.LocalContentColor
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.SelectableChipColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.ColorPainter
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.orange.ods.R
+import com.orange.ods.compose.component.utilities.OdsImageCircleShape
+import com.orange.ods.compose.text.OdsTextBody2
+
+/**
+ * <a href="https://system.design.orange.com/0c1af118d/p/81aa91-chips/b/13c40e" target="_blank">ODS Chips</a>.
+ *
+ * Chips are small components containing a number of elements that represent a calendar event or contact.
+ *
+ * Use Accompanist's [Flow Layouts](https://google.github.io/accompanist/flowlayout/) to wrap chips to a new line.
+ *
+ * @param text Text to display in the chip.
+ * @param onClick called when the chip is clicked.
+ * @param modifier Modifier to be applied to the chip
+ * @param outlined If set to true, a border will be drawn around the chip.
+ * @param enabled When disabled, chip will not respond to user input. It will also appear visually
+ * disabled and disabled to accessibility services.
+ * @param selected Highlight the chip and display a selected icon if set to true.
+ * @param leadingAvatar Optional avatar at the start of the chip, preceding the content text.
+ * @param leadingAvatarContentDescription Content description associated to the avatar.
+ */
+@ExperimentalMaterialApi
+@Composable
+fun OdsFilterChip(
+    text: String,
+    onClick: () -> Unit,
```

Comment:
Yes I think...

I fixed this problem adding a disabled interaction source to avoid ripple effet on click
```kotlin
interactionSource = if (enabled) remember { MutableInteractionSource() } else remember { DisabledInteractionSource() },
```
and changing the `onClick` value
```kotlin
onClick = if (enabled) onClick else emptyAction,
```